napari/layers/_shapes_layer/model.py

```diff
@@ -1262,8 +1198,7 @@ def on_mouse_press(self, event):
         elif self.mode in ([Mode.ADD_RECTANGLE, Mode.ADD_ELLIPSE,
                             Mode.ADD_LINE]):
             # Start drawing a rectangle / ellipse / line
-            rescale = self._get_rescale()
-            size = self._vertex_size*rescale/4
+            size = self._vertex_size*self.scale_factor/4
```

Comment:
im just gonna stop here. maybe we'll fix at a later point. all we really need is someone to run `autopep8`

you should hook up pep8/flake8 on your IDE to resolve these issues prior to push

Reply:
ok, i tried to fix all the rest of these - it is strange i didn't get any pep8 errors on these things